Riot/Modules/Room/VoiceMessages/VoiceMessageController.swift

```diff
@@ -240,13 +269,20 @@ public class VoiceMessageController: NSObject, VoiceMessageToolbarViewDelegate,
             return
         }
         
+        loadDraftRecording()
+        
+        updateUI()
+    }
+    
+    func loadDraftRecording() {
+        guard let temporaryFileURL = temporaryFileURL else {
+             return
+        }
         audioPlayer = mediaServiceProvider.audioPlayerForIdentifier(UUID().uuidString)
```

Comment:
Can we maybe use the room id here instead of a random uuid? I think that might make it save the playback position as well.